frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1,13 +1,12 @@
-local InputContainer = require("ui/widget/container/inputcontainer")
-local ReaderPanning = require("apps/reader/modules/readerpanning")
-local Screen = require("device").screen
 local Device = require("device")
-local Input = require("device").input
+local InputContainer = require("ui/widget/container/inputcontainer")
 local Event = require("ui/event")
+local ReaderPanning = require("apps/reader/modules/readerpanning")
 local UIManager = require("ui/uimanager")
 local logger = require("logger")
 local _ = require("gettext")
-
+local Input = require("device").input
```

Comment:
Sorry, I didn't notice this before and it's not your fault but would you mind changing this to `Device.input`?